airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,359 @@
+| updated_at | timestamp with time zone | yes | current_timestamp | `current_timestamp` is implemented in most databases. |
+
+## Tables
+
+### `STANDARD_WORKSPACE`
+
+YAML [reference](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardWorkspace.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| customer_id | char(36) | | | Not a foreign key. |
+| name | varchar(255) | yes | | |
+| slug | varchar(255) | yes | | |
+| email | varchar(255) | | | index |
+| initial_setup_complete | boolean | yes | false | |
+| anonymous_data_collection | boolean | | true | |
+| news | boolean | | false | | |
+| security_updates | boolean | | false | |
+| display_setup_wizard | boolean | | true | Artem suggested to default this field to `true`. |
+| tombstone | boolean | | false | index |
+
+### `WORKSPACE_NOTIFICATION`
+
+YAML reference:
+- [Notification](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/Notification.yaml)
+
+This table has many-to-one relationship with the `WORKSPACE` table. One workspace can have multiple notifications.
+
+This is the base table for notifications. Each notification type will have its own child table (e.g. `SLACK_NOTIFICATION`).
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| notification_type | varchar(50) | yes | | Index. String enum: "slack". |
+| send_on_success | boolean | | false | index |
+| send_on_failure | boolean | | false | index |
+
+### `SLACK_WORKSPACE_NOTIFICATION`
+
+YAML reference:
+- [SlackNotificationConfiguration](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SlackNotificationConfiguration.yaml)
+
+This table has one-to-one relationship with the `WORKSPACE_NOTIFICATION` table. Each `SLACK_WORKSPACE_NOTIFICATION` belongs to one `WORKSPACE_NOTIFICATION`.
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| notification_id | char(36) | yes | | Foreign key. Unique index to enforce 1:1 mapping. |
+| webhook | varchar(2048) | yes | | |
+
+### `SOURCE_DEFINITION`
+
+YAML reference:
+- [StandardSourceDefinition](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardSourceDefinition.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | | |
+| docker_repository | varchar(255) | yes | | Index (ideally this should be unique, but we have duplicates for marketing purpose). |
+| docker_image_tag | varchar(50) | yes | | |
+| documentation_url | varchar(255) | yes | | |
+| icon | varchar(63) | | | |
+
+### `SOURCE_CONNECTION`
```

Comment:
I think we can just call this SOURCE. SOURCE_CONNECTION was a name we used before we settled on the current nomenclature.